java/core/src/test/org/apache/orc/TestOrcFilterContext.java

```diff
@@ -252,23 +251,23 @@ public void testRepeatingVector() {
     assertTrue(OrcFilterContext.isNull(vectorBranch, 1));
     assertTrue(OrcFilterContext.isNull(vectorBranch, 2));
   }
-  
```

Comment:
May I ask why Checkstyle complain about this? Or, is it deleted accidentally?

Reply:
This was accidental deletion, added it back.
I really admire the way you catch things 👍